lec09-Randomization-tests.Rmd

```diff
@@ -53,7 +53,7 @@ library(dplyr)
 library(EcoSimR)
 ```
 
-```{r}
+```{r message=FALSE, warning=FALSE}
```

Comment:
Another suggestion. If you want to have `message=FALSE, warning=FALSE` for all code chunks, you can do that in one line. By including the below code in the very first code chunk at the top, you can have the chunk option set globally for the document:

```
knitr::opts_chunk$set(warning = FALSE, message = FALSE)
```

Reply:
Oh sweet I didn't know that! Thanks for the heads up.